packages/client/src/exampleTypes.js

```diff
@@ -194,7 +194,8 @@ export default F.stampKey('type', {
     },
     defaults: {
       field: null,
-      range: [0, 1000],
+      min: 0,
+      max: 1000,
```

Comment:
I know this was here already but why have a `max` default? I think a range unbounded on its maximum value makes more sense.